cluster/saltbase/salt/kube-apiserver/initd

```diff
@@ -38,6 +38,9 @@ DAEMON_USER=kube-apiserver
 #
 do_start()
 {
+        # Raise the file descriptor limit - we expect to open a lot of sockets!
+        ulimit -n 65536
```

Comment:
Since we increate the limit to 1.000.000 for etcd anyway (as for everything that is running inside Docker containers), maybe it makes sense to change it to 1.000.000 here too?